examples/toy-model-stochastic-degradation.ipynb

```diff
@@ -0,0 +1,190 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "mean = model.deterministic_mean(k, times)\n",
+    "variance = model.deterministic_variance(k, times)\n",
+    "std_dev = np.sqrt(variance)\n",
+    "\n",
+    "plt.plot(times, mean, '-', label = 'mean')\n",
+    "plt.plot(times, mean + std_dev, '--', label = 'standard deviation upper bound')\n",
+    "plt.plot(times, mean - std_dev, '--', label = 'standard deviation lower bound')\n",
+    "plt.legend(loc = 'upper right')\n",
```

Comment:
Can you add labels to both axes here? And make sure all labels are consistent across plots (all three in this notebook).